config/types.ts

```diff
@@ -79,6 +79,12 @@ export interface Conference {
   Keynotes: Session[]
 }
 
+export enum SoldOutOptions {
```

Comment:
TicketOptions or TicketStatus as a name for the enum could provide more clarity

Reply:
Totally agree, was being super lazy. Will fix 👍